From 50bce37120e3a1f448e62dd88b0b8cce1f36d35b Mon Sep 17 00:00:00 2001
From: Nils Philippsen <nils@redhat.com>
Date: Wed, 16 Mar 2011 10:59:09 +0100
Subject: [PATCH] patch: man-encoding

Squashed commit of the following:

commit 54a279b424c39c0ab658acacb5e2c861b6f291a0
Author: Nils Philippsen <nils@redhat.com>
Date:   Wed Mar 16 10:52:45 2011 +0100

    don't use ISO-8859-1 encoding in man pages
---
 doc/sane-hpljm1005.man |    2 +-
 doc/sane-p5.man        |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/sane-hpljm1005.man b/doc/sane-hpljm1005.man
index af6588f..ff1006a 100644
--- a/doc/sane-hpljm1005.man
+++ b/doc/sane-hpljm1005.man
@@ -34,4 +34,4 @@ support dynamic loading).
 .BR sane\-usb (5),
 
 .SH AUTHOR
-Philippe Rétornaz <couriousous@mandriva.org>
+Philippe R\['e]tornaz <couriousous@mandriva.org>
diff --git a/doc/sane-p5.man b/doc/sane-p5.man
index b5ecbba..860a019 100644
--- a/doc/sane-p5.man
+++ b/doc/sane-p5.man
@@ -133,7 +133,7 @@ St\['e]phane Voltz <stef.dev@free.fr>
 
 .SH "CREDITS"
 Support for the Prima PagePartner has been made possible thank to an hardware donation
-by Sébastien Lange.
+by S\['e]bastien Lange.
 
 .SH "BUG REPORTS"
 If something doesn't work mail sane-devel@lists.alioth.debian.org or use the bug
-- 
1.7.4

